src/scheduler/workers/processListsBeforeAndDuringStart/helpers.ts

```diff
@@ -14,11 +14,11 @@ export function formatDate(date: Date = todayDateString) {
   return date.toLocaleString("en-gb", options);
 }
 
-export function isEmailSentBefore(
-  audit: Audit | undefined,
+export function hasDataContainingReference(
```

Comment:
The function name should probably revert back - the code doesn't actually check if the audit or event has `reference`?